README.md

```diff
@@ -1529,6 +1549,63 @@ $ ./bin/neo-go contract invokefunction -r http://localhost:20331 -w my_wallet.js
 $ ./bin/neo-go contract invokefunction -r http://localhost:20331 -w my_wallet.json 9042814f07d65d2b835fa1f07d21c22c6e1cbdf7 delete my_second_domain -- NbrUYaZgyhSkNoRo9ugRyEMdUZxrhkNaWB
 ```
 
+## Workshop. Part 5
+
+In this part we'll take a look at the example of a decentralized application for the
+Neo ecosystem written in Go. The example demonstrates how to use NeoGo RPC Client
+and a various set of helpers to work with wallets, deploy and invoke smart
+contracts, retrieve data from the blockchain, handle blockchain events and create
+auto-generated smart contract RPC bindings.
+
+#### Step #1
+
+Ensure that private network created at the [Preparation part](#workshop-preparation)
+step is up and running. We'll also need a `NbrUYaZgyhSkNoRo9ugRyEMdUZxrhkNaWB`
+account from the [wallet](https://github.com/nspcc-dev/neo-go-sc-wrkshp/blob/master/my_wallet.json)
+with some GAS on it. Take a look at the dApp example: [dApp.go](./dApp/dApp.go).
+The example includes all basic API usages you need to be aware of to start your
+dApp development for Neo:
+* [JSON RPC client](https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/rpcclient) creation, initialization and example of simple Neo JSON RPC APIs usage.
+* Neo wallet and account managing with [NeoGo `wallet` package](https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/wallet).
+* [Extensions](https://github.com/nspcc-dev/neo-go/blob/master/docs/rpc.md#extensions) offered by NeoGo JSON RPC server and supported by the NeoGo RPC client.
+* NeoGo JSON RPC server [web-socket extension](https://github.com/nspcc-dev/neo-go/blob/master/docs/rpc.md#websocket-server). [Web-socket JSON RPC Client](https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/rpcclient#WSClient) creation, initialisation and usage example. 
+* [`unwrap` helper package](https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/rpcclient/unwrap) for test invocation results unwrapping.
+* NeoGo JSON RPC [Notification subsystem](https://github.com/nspcc-dev/neo-go/blob/master/docs/notifications.md) usage example. Subscribe for a various set of blockchain node events and receive notifications over web-socket channel.
+* NeoGo `Invoker` package usage example. Perform test invocation of smart contract, script or verification method with the powerful [Invoker API](https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker).
+* How to work with witnesses and [witness scopes](https://neospcc.medium.com/thou-shalt-check-their-witnesses-485d2bf8375d).
+* [Historic invocations functionality](https://github.com/nspcc-dev/neo-go/blob/master/docs/rpc.md#invokecontractverifyhistoric-invokefunctionhistoric-and-invokescripthistoric-calls) provided by NeoGo RPC server and client as an extensions.
```

Comment:
"as extensions" or just "server and client extensions"

Reply:
Fixed.